rskj-core/src/test/java/org/ethereum/core/BlockHeaderExtensionV2Test.java

```diff
@@ -247,9 +247,9 @@ void testBaseEventWithAllOnesBytes() {
 
     @Test
     void testBaseEventWithMaxSize() {
-        // Test with maximum practical size (64KB)
-        byte[] maxSizeValue = new byte[65536];
-        for (int i = 0; i < 65536; i++) {
+        int numberOfBytes = 128;
```

Comment:
Perhaps we could also have a test showing it fails to set a base event with more than 128 bytes, what do you think?